cli/cmd/vuln_container_list_assessments.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/lacework/go-sdk/api"
 	"github.com/lacework/go-sdk/internal/array"
+	"github.com/lacework/go-sdk/lwtime"
```

Comment:
Love this!

Reply:
I love our `lwtime` Go package. I really do!

<img src="https://media3.giphy.com/media/fkD36jhiqzJ9m/giphy.gif"/>